### PR TITLE
Updated .gitignore

### PR DESCRIPTION
**Pull request by BondarenkoArtur**
I offer to update .gitignore file because now git adds unneeded files

### CODE REVIEW

**Comment by DeForce** [comment]
Just don't add them to git?
As for now I don't see any of those files that you have listed in source.
Are you sure we need to add them?

**Comment by BondarenkoArtur** [comment]
I have in `logs` folder two files `2016-10-05.txt` and `2016-10-06.txt` with chat log, and empty `df.txt`
In `conf` folder I got database file `levels.db.random` with one table UserLevels
DB table UserLevels

Probably you ignore this files in pycharm, not in git, try `git status` in root project folder

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 *.pyc
-.idea/*
\ No newline at end of file
+.idea/*
+/logs
+*levels.db.random
```
Comment (DeForce) [inline .gitignore]: Change to /conf/levels.db*

**Comment by DeForce** [comment]
Added comments

**Comment by BondarenkoArtur** [comment]
Made update for levels.db

**Comment by DeForce** [comment]
#67

**Comment by DeForce** [comment]
Rebase and squash.

**Comment by BondarenkoArtur** [comment]
Updated

**Pull request merged**